lib/api/v3/work_packages/form/schema_allowed_priorities_representer.rb

```diff
@@ -35,6 +35,15 @@ module V3
     module WorkPackages
       module Form
         class SchemaAllowedPrioritiesRepresenter < Decorators::SchemaAllowedValuesRepresenter
+          def initialize(model, context = {})
+            super(model,
+                  'Priority',
+                  I18n.t('activerecord.attributes.work_package.priority'),
+                  true,
+                  true,
+                  context)
```

Comment:
@NobodysNightmare, without looking it up, please tell me what the two ```true```s do. This interface is way to large.